model/telemetry/registry.yaml

```diff
@@ -81,3 +81,57 @@ groups:
         brief: >
           The version string of the auto instrumentation agent or distribution, if used.
         examples: ["1.2.3"]
+      - id: telemetry.sdk.processor.type
```

Comment:
it's not possible to define attributes in the metric. Attribute definition must happen in the registry and can be referenced from specific telemetry signals - check out https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#1-modify-the-yaml-model and https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/README.md